keyboards/mntre_v3/readme.md

```diff
@@ -0,0 +1,27 @@
+# MNT Reform USB Keyboard (Standalone)
+
+![MNT Reform USB Keyboard (Standalone)](https://imgur.com/a/Vr1vvvv)
```

Comment:
```suggestion
![MNT Reform USB Keyboard (Standalone)](https://i.imgur.com/pXDDan6h.jpeg)
```

Reply:
thank you for the suggestion i have edited the link